products/redhat.md

```diff
@@ -11,32 +11,63 @@ releasePolicyLink: https://access.redhat.com/support/policy/updates/errata
 activeSupportColumn: true
 releaseDateColumn: true
 releaseColumn: false
-sortReleasesBy: 'release'
+sortReleasesBy: 'cycleShortHand'
 releaseLabel: "RHEL __RELEASE_CYCLE__"
 releases:
   - releaseCycle: "9"
     release: 2022-05-18
     support: 2027-05-31
     eol: 2032-05-31
     latest: "9.0"
+    cycleShortHand: 10
+  - releaseCycle: "9 (ELS)"
```

Comment:
Let's mark these as `lts: true` with a `LTSLabel` like in sles.md?

Reply:
To clarify, what you want is to add this: 

```
LTSLabel: "<abbr title='Extended Life Cycle Support'>ELS</abbr>"
```

And then add `lts: true` to every ELS?

Reply:
Yup.